tasks/cluster_agent.py

```diff
@@ -123,3 +127,38 @@ def image_build(ctx):
     shutil.copy2(latest_file, "Dockerfiles/cluster-agent/")
     ctx.run("docker build -t {} Dockerfiles/cluster-agent".format(AGENT_TAG))
     ctx.run("rm Dockerfiles/cluster-agent/datadog-cluster-agent")
+
+
+@task
+def add_prelude(ctx, new_version):
+    """
+    Add prelude releasenote for a new minor version of the Datadog Cluster Agent.
+    """
+    res = ctx.run("""echo 'prelude:
+    |
+    Release on: {1}'\
+    | EDITOR=tee reno -d releasenotes/cluster-agent new prelude-release-{0} --edit""".format(new_version, date.today()))
+
+    new_releasenote = re.search(r"(releasenotes/cluster-agent/notes/prelude-release-{0}-[\w]+.yaml)".format(new_version), res.stdout).groups()[0]
+    ctx.run("git add {}".format(new_releasenote))
+    ctx.run("git commit -m \"Add prelude for {} release\"".format(new_version))
+
+
+@task
+def update_changelog(ctx, new_version):
```

Comment:
This doesn't handle bugfix release (like said in my other comment).

We're in the process of updating the `update_changelog` method for the agent to work on windows. Their is a lot of code duplication between `task/cluster_agent.py` and `task/release.py`. If you want we could merge both command set in another PR ?